static/src/js/reader/widgets/morpheus.vue

```diff
@@ -70,9 +70,10 @@ export default {
   methods: {
     fetchData() {
       const word = this.selectedWord;
+      const lang = this.$store.getters['reader/text'].metadata.lang;
```

Comment:
It would be best to make this a computed property. This will ensure the Vue reactivity system will correctly handle dependencies in the code. Ideally, you create a `text` property around `this.$store.getters['reader/text']`.